### PR TITLE
fix: prevent mobile tabbar from opening new window for Master Logs [DET-4654]

### PR DESCRIPTION
**Pull request by hkang1**
## Description

2020-12-09 at 12 23 PM

## Test Plan

In mobile view, check that Master Logs (under the overflow button) doesn't open a new tab

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/components/NavigationTabbar.tsx [hunk 1]
```diff
@@ -51,6 +52,11 @@ const NavigationTabbar: React.FC = () => {
     setIsShowingOverflow(false);
   }, []);
 
+  const handlePathUpdate = useCallback((path) => {
+    history.push(path);
```
Comment (hamidzr) [inline webui/react/src/components/NavigationTabbar.tsx:56]: suggestion: could we use handlePath from `routes/utils`?

**Review by hamidzr — COMMENTED** [review]
Looks good I have a small comment

File: webui/react/src/components/NavigationTabbar.tsx (hunk 1 above)
Comment (hkang1) [reply webui/react/src/components/NavigationTabbar.tsx:56]: nice good call, I keep forgetting stuff like this lol

**Review by hamidzr — APPROVED** [review]
:rocket: :ship:

File: webui/react/src/components/NavigationTabbar.tsx (hunk 1 above)
Comment (hamidzr) [reply webui/react/src/components/NavigationTabbar.tsx]: that was quick! thx

**Comment by hamidzr** [comment]
nit: probably a smaller commit summary would be more consistent with the rest

**Pull request merged**